tipocket-ctl/tpctl/deploy.py

```diff
@@ -15,10 +15,22 @@ def __init__(self, env, feature, case, binary, image, params):
         self.image = image
         self.params = params
 
-        self._argo_workflow_filepath = \
-            f'{env.dir_root}/{case.name}-{feature}.yaml'
+        if params['cron']:
```

Comment:
we have several piece of code 
```
if params['cron']:
    xxx
```
I think we can simplify it to
```
self._is_cron = params['cron'] is True

if self._is_cron:
    xxx
```

Reply:
Yes.